modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc

```diff
@@ -1,115 +1,165 @@
-5.  Restart Apache
+== Description of Steps
+
+The following steps are neccesary to move the data directory.
+
+. Stop Apache
+. Enable mainenance mode for your instance
+. Use `rsync` to sync the files from the current to the new directory
+. Double-check the xref:installation/installation_wizard.adoc#post-installation-steps[directory permissions]
+  on the new directory
+. Change the ownCloud configuration to point to the new data directory
+. Disable mainenance mode for your instance
+. Restart Apache
+
+Look at each section below for a detailed description.
+
+== Apache and rsync
 
-To save time, here’s the commands which you can copy and use:
+To save time, here are the commands which you can copy/paste for Apache and rsync:
 
 [source,console]
 ----
-apachectl -k stop
-rsync -avz /var/www/owncloud/data /mnt/owncloud
-ln -s /mnt/owncloud /var/www/owncloud/data
-apachectl -k graceful
+sudo service apache2 stop
+
+sudo service apache2 start
+
+sudo rsync -avz /var/www/owncloud/data /mnt/owncloud
 ----
 
-NOTE: If you’re on CentOS/Fedora, try `systemctl restart httpd`. If you’re on Debian/Ubuntu try
-`sudo systemctl restart apache2` To learn more about the systemctl command, please refer to
-https://www.digitalocean.com/community/tutorials/systemd-essentials-working-with-services-units-and-the-journal[the systemd essentials guide].
+NOTE: Check your commands how to start or stop your webserver if you are not on Ubuntu/Debian.
+
+== Enable and Disable Maintenace Mode
+
+It is necessary to enable maintenace mode to avoid that cron jobs can run.
+
+To enable maintenace mode of your instance run the following command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} maintenance:mode --on
+----
+
+To disable maintenace mode of your instance run the following command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} maintenance:mode --off
+----
 
 == Fix Hardcoded Database Path Variables
 
-=== Update the oc_storages table
+Open a database command line client to enter database commands and activate your owncloud database.
 
-If you want to manually change the location of the data folder in the
-database, run the SQL below:
+[source,sql]
+----
+use owncloud;
+----
+
+=== Update the oc_storages Table
+
+Run the SQL below:
 
 [source,sql]
 ----
-UPDATE oc_storages SET id='local::/mnt/owncloud'
+UPDATE oc_storages SET id='local::/mnt/owncloud/data/'
   WHERE id='local::/var/www/owncloud/data/';
 ----
 
-=== Update the oc_accounts table
+=== Update the oc_accounts Table
 
 You next need to update the `home` column in the `oc_accounts` table.
 This column contains the absolute path for user folders, e.g.,
-`/mnt/data/files/admin`. Assuming that the new home directory is:
-`/mnt/data/files/super-admin`, and that the user’s id is 1, you could
-change it using the following SQL statement:
+`/mnt/owncloud/data/files/my_user`.
```

Comment:
This stuff seems odd (old and new).
e.g. I have a local install with something like `/var/www/owncloud/data/anne/files` `/var/www/owncloud/data/bbob/files` etc. The UID is directly under the `data`  folder, and then the user has a `files` folder under that???

Reply:
You are right, I will fix that